wsclient/src/main/java/org/hpccsystems/ws/client/antlr/EclRecord.g4

```diff
@@ -104,6 +105,10 @@ xmldefaultval:
     'XMLDEFAULT' OPAREN STRING CPAREN
 ;
 
+setdefaultval:
+    ('DEFAULT' OPAREN OSQUARE STRING CSQUARE CPAREN | 'DEFAULT' OPAREN SETSTRING CPAREN)
```

Comment:
@jchambers-ln what's the difference between 
'DEFAULT' OPAREN OSQUARE STRING CSQUARE CPAREN
and 
'DEFAULT' OPAREN SETSTRING CPAREN

Reply:
'DEFAULT' OPAREN OSQUARE STRING CSQUARE CPAREN matches DEFAULT ([STRING])
'DEFAULT' OPAREN SETSTRING CPAREN matches DEFAULT(STRING)

I tried several variations this is the one that worked, if you have someone that knows antlr better I'm open to suggestions.